shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2677,38 +2716,28 @@ \subsubsection{Monetary Expansion}
 \item
   rising transaction fees when more and more people use the system and
 \item
-  higher exchange rates from ADA to USD when the system become more
+  higher exchange rates from ada to USD when the system become more
   valuable.
 \end{itemize}
 
 Note that the fees that have been collected during the bootstrap era,
 where all nodes have been provided by IOHK, Emurgo, and the Cardano
 Foundation, have not been paid out. Those fees have reduced the amount
-of ADA currently in existence, but they did not change
+of ada currently in existence, but they did not change
 \(T_\infty\). Effectively, those fees will be distributed amongst
 future stakeholders via monetary expansion.
 
```

Comment:
Should we give a precise definition of what "ada in existence" means? Is at the sum over all outputs of the current UTxOs?

Reply:
Maybe replace "existence" by "circulation"? We don't have a T pot and rewards pot yet, currently we only have the ada in circulation, right?